src/main/webapp/help-skipPolling.html

```diff
@@ -0,0 +1,5 @@
+<div>
+    (Optional) If checked the jenkins job will not wait for the provison operation to complete
+    <strong> Note:- The newly provisioned VDB would be in UNKNOWN state
```

Comment:
`...will be in a temporary UNKNOWN state.`

Reply:
(Optional) If checked, the Jenkins step will not wait for the provisioning to complete.